syntax = "proto3";

package terragen;

message BlockUpdateBroadcastModel {
  string id = 1;
  bytes data = 2;
}